core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2061,9 +2061,8 @@ class SparkContext(config: SparkConf) extends Logging {
    * Cancel a given job if it's scheduled or running.
    *
    * @param jobId the job ID to cancel
-   * @throws InterruptedException if the cancel message cannot be sent
+   * @note Throws `InterruptedException` if the cancel message cannot be sent
```

Comment:
Hm.. interesting. I haven't looked into this deeper but it seems it fails anyway.

```
[error] .../java/org/apache/spark/SparkContext.java:1150: error: exception not thrown: java.lang.InterruptedException
[error]    * @throws java.lang.InterruptedException if the cancel message cannot be sent
[error]              ^
```